### ISSUE TITLE
Readme needs setup instructions

### ISSUE THREAD

**Issue by codestu12** [Home-Is-Where-You-Hang-Your-Hack/sensor.goveetemp_bt_hci#2]
Readme should include which linux packages are required, ie:
sudo apt-get install bluez-hcidump

I was also required to set permissions to hcidump so it would run without requiring sudo, with:
sudo setcap 'cap_net_raw,cap_net_admin+eip' `which hcidump`

You should probably also consider adding a section on installing with HACS

**Comment by Thrilleratplay** [comment]
HACS is how I currently have it installed as an add-on repo on my own Home Assist install but wanted to finalize a few things and get feed back from ~~suckers~~ brave souls willing to test before submitting to a wider audience.

You are right though; I do need to make sure that bluez hcidump and hcitool along with necessary permissions are explicitly stated.

**Comment by codestu12** [comment]
I installed it through HACS as well. It failed the first time because I didn't have hcidump. I installed successfully after I fixed that though. The log message was pretty clear as to what happened.

I took me a while to figure out that hcidump required sudo or modified permissions because the result of the process call is blank because you redirected stderr to /dev/null. It might be worth calling hcitool and hcidump once on setup or hass boot with stdout redirected to /dev/null and stderr redirected to stdout to see if anything comes up and then generate a notification or error level log entry with the contents of the process call result.

I have it running fine with one sensor at the moment. I am about to buy a few more.
